config/generic_domains.php

```diff
@@ -33,3 +33,826 @@
         'type' => 'text'
     ],
 ]);
+
+// All available TLDs
```

Comment:
The TLD list contains several wildcard TLDs, however the Domains plugin does not parse them, so `.co.com` does not match `.*.com`, I think it would be necessary to update the Domains plugin to add support for wildcard TLDs.

Reply:
For now we'll put off handling wildcard.  We'll need to update the order form to match domain searches to wildcard tlds.